src/OneExplorer/OneConfigSetting.ts

```diff
@@ -0,0 +1,250 @@
+import * as vscode from "vscode";
+
+import { Locator, LocatorRunner } from "./ArtifactLocator";
+
+export type OneCfg = {
+  "one-import-tflite": CfgOneImportTflite;
+  "one-import-onnx": CfgOneImportOnnx;
+  "one-import-tf": CfgOneImportTf;
+};
+type CfgOneImportTflite = any;
+type CfgOneImportOnnx = any;
+type CfgOneImportTf = any;
+
+export class OneConfigSetting {
+  static baseModelsLocatorRunner: LocatorRunner;
+  static productsLocatorRunner: LocatorRunner;
+  // TODO: make sections for updateBaseModelField method
+
+  static getBaseModelsLocatorRunner(): LocatorRunner {
```

Comment:
Q. Why should it be Singleton? I think it can simply be a class and function.

Reply:
A. I think that if config use same setting, config use same locatorRunner. If it has problem, I will change it to simple class function.

Reply:
Making something Singleton is to use a same object across the whole project, it would be for sustaining a certain context or something. In this case, I think it's enough to go with a simple class. :-D